src/view/com/composer/Composer.tsx

```diff
@@ -211,6 +225,7 @@ export const ComposePost = observer(function ComposePost({
         onStateChange: setProcessingState,
         knownHandles: autocompleteView.knownHandles,
         langs: store.preferences.postLanguages,
+        tags,
```

Comment:
Outline tags are posted to API here, inline are contained in the raw text and detected later